src/app/page-not-found/page-not-found.component.html

```diff
@@ -0,0 +1,7 @@
+<div id="contentWrapper">
+  <div class="centerContentWrapper">
+    <h2>404 - Page Not Found</h2>
+    <p>Sorry, we can't find the page you're looking for!</p>
+    <p>You can return to the previous page, go back to our <a href="/">homepage</a>, or <a href="/contact">contact us</a> for support.</p>
```

Comment:
If these hrefs refresh the page, use routerLinks instead

Reply:
Fixed!